addons/copy-message-link/userscript.js

```diff
@@ -0,0 +1,48 @@
+export default async function ({
+  addon,
+  global,
+  console
+}) {
+  if (!addon.tab.clientVersion) return;
+  await addon.tab.waitForElement("div.comment", {
+    markAsSeen: true
+  });
+  if (addon.settings.get('reply')) {
+    document.querySelectorAll('div.comment').forEach((elem) => {
```

Comment:
Something better to do here would to put the `await addon.tab.waitForElement` and set it to a variable, then you can just change that variable. You can read more about addon.tab.waitForElement in the wiki